test/component_fixtures/hello_world.ex

```diff
@@ -0,0 +1,5 @@
+defmodule HelloWorld do
+  @moduledoc false
+
+  use Wasmex.Components.Component, wit: "test/component_fixtures/hello_world/hello-world.wit"
+end
```

Comment:
love it! would be great to see imported functions being support soon 🙌 

oh and is it possible to move this module just above the test where we use it? (like an inlined module). this would make the test way easier to read IMO (since we don't have to jump files)